src/main/groovy/org/mockito/release/internal/gradle/ReleaseConfigurationPlugin.java

```diff
@@ -19,6 +29,8 @@
 
     private ReleaseConfiguration configuration;
 
+    private static final String CONFIG_FILE_RELATIVE_PATH = "gradle/shipkit.gradle";
```

Comment:
Consider making it public and reusing in the places where refer to this file using String literal. It would be slightly more cleaner I think.